sdks/open-format/src/core/contract.ts

```diff
@@ -454,6 +454,12 @@ export async function deploy({
 
     const customMetadata = nft.metadata ?? {};
 
+    console.log(nftStorageToken);
+
+    const nftStorage = new NFTStorage({
+      token: nftStorageToken || defaultNFTStorageToken,
```

Comment:
```suggestion
      token: nftStorageToken,
```

After we remove the default token.